Python/notify_me/README.md

```diff
@@ -0,0 +1,40 @@
+# Notify Me
+
+* supported OS: windows 10
+* Push desktop notification every 4 hours to notify you the TODO items
+
+
+
+### Pre-requisites
+1. install python in your windows 10
+2. use pip to install plyer
+
+```
+$ pip install plyer
+```
+
+### How to start
+
+Double click notify-me.bat to start the program
+
+### How to run every time system boot
+1. Create a short-cut of notify-me.bat under notify_me dirctory
+2. press win + r and enter "shell:startup":
+  ![run](https://lh3.googleusercontent.com/1FL17gW0BCTsxSOWMOg6B40J-jZOK7Q1mc745ZTBRePsfgT3RT9x3xA32deC4hmwjiUh9Kxe-3eNCdQFVoVR90kOgCG11IxilIRuCvWTcKG6RIz-DI7ahGmENoCj1Ontt5IT1_AkXpWMBoHfLng4U7N5KXVt_R76gFAAFyMC6fT7MNEWJh5T7ET4WFy0tfve6mONr1SbNInfuu9ImH0nKtpsBcEHJn-KkbqinKjIyFVMIw0Emw5ASWNYMrfjgkLv-dmtNMUnMFl55H3mxvK4oAZhEPgccWrfuEw3-dc3PeDRcyjYvEMr_Ymn5qzL-QzbN2kNLHubeO44a2DNgiL_mlVdxEIUgNxNiQXe167qAIU2AhufeLKv8cJeJvdohZjJwMrZd3zB7iyHSmzVQQZs0VLGa02AW1VD0gieCK37y3wy5LLmFEe_5FvsKajR6Gyb2XZ9kfSHdDqaeQ8yq1quzDwitq9Q_2FNqqISNuyvYewaZ3Y-YfnnwEoaXbU0PDHR7xOM1zeqCUxXUNzdfLnat2_43EVkeqTxALUy7g3_8rhiGdCJDRk3zHZxYUMHYEahTMvbe624yrmmUMlW-hNmYhWMYVVhY-BikjuagvF4DDQjrNZUHzNHx8gSHuFrKt4eJgcYSHJEmsPYv99GSPQDtaigTk-74KbB3wjxBnBiqiIdkPKTb7s4T1TaQiTWDnA=w824-h478-no?authuser=0)
```

Comment:
Images are not getting rendered

Reply:
@kaustubhgupta 
In my branch, the images are rendered normally as below (it is under google link):
![run](https://lh3.googleusercontent.com/dludVNz2DPKvG2ER2vtSzNU22mag945nQwVL6N1OEiLkrpV-KeelZwCRh30xqW3vavbJ4I-te1GSUFxrA4c5lAs4tZ34xK__EIwxxlGhooOQj6DgFti6fc8GDr57LHDXvQg3M62a0DrbfZTxSZIwCTsX2fip_zqoo6KSjLwFiC-kjji2ic55ZV54HWoGr1qHYj39npdUYTwmDLFdR8VocAlLJyLZdAmiXBC-ue7TFa7JFOv-CG0E_rXO4emMMxJ9yc5foxckU_anKbfC5MXQmkz5pDtjsPZ_-ymSlLjAa-jsVHeZPgR-fqgunjlvEMTCp3iYLF20FyCuheqbbc8uqasvewscEI8VnxJb7ZqaIyqRy3KGXhUOqwDlp1IwUCkoMAZHwnuoVd7bSbP_du7q_oUsxLqSyrufGUS2ZU2qtnUkhANUdPm-QDcPvw124R-z7JdjPlOLOgvMtIUsl1dX9ybuj70GaParoJyG1XqzPslqpFDv_b2pwzDJasOWgDNWuvETApLa-Sul8a3WJ8SnU8kGzWYveu1IV5j9i8a17CpvqBu6ENDYiIEKFekBVYFz-fpFUwhwWKareTnSTJcjnEU2L7EBR4kOxcvfB7qY2XFAvwbnIp2BUO7P-0olEHY_aAsdcc03a0l4n_eRTl0yNHhI025qh_p9BD_3m3GjFgLvUyJHZicCBaFl9H9-vRE=w328-h275-no?authuser=0)

I am not sure why, might I ask your kind help to double confirm again?

Reply:
![image](https://user-images.githubusercontent.com/43691873/96089046-e6f34600-0ee3-11eb-8088-6ef710e0880a.png)
I am not able to see images in your branch too

Reply:
@kaustubhgupta 
Understood, it is wired. I will change to other cloud storage to see if it can be fixed.

Reply:
I have a solution, simply take the screenshot using the snipping tool in windows, you will have an option to copy the image, do that and simply paste it in the readme. It will be automatically uploaded in user-images.githubusercontent.com and you will be able to see the preview instantly.

Reply:
Thanks for the useful suggestion! I had modified it.
Please kindly help to double confirm if it is ok.